### ISSUE TITLE
[A11y]Keyboard focus is lost after "Delete" button.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7877]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1062703

Steps to Reproduce:
1.Launch application using provided URL.(https://www.nuget.org/)
2. Navigate to "Sign in" control and hit enter.
3. Navigate to "Sign in with Microsoft" control and hit enter.
4. Select “API Keys” from the Profile Dropdown and hit enter.
5. Navigate to "Create" button and hit enter.
6. Enter Values in "Key Name" and "Glob Pattern" field and hit enter on Create button.
7. Navigate to "Manage" control and hit enter.
8. Navigate to the "delete"  button.
9. Check the keyboard focus after delete button.

Actual:
The Keyboard focus is lost after "Delete" button. The keyboard focus is not visible anywhere on the screen after the  delete button focus. 

Expected:
The focus should be visible on the next control after the delete button.

Notes:
This is indicating that keyboard focus is lost after the "Delete" button is interacted. Tabbing afterwards correctly brings focus to the following element, so it appears that the focus is not so much lost as it is not correctly moved forward. We need to focus the next element when delete is pressed so the element with focus is visible. Possibly related stackoverflow https://stackoverflow.com/questions/7208161/focus-next-element-in-tab-index

**Comment by ryuyu** [comment]
Focus is not lost; tab continues where the page was left.
It is not expected that removed elements retain focus, and the removal is only after an explicit removal action.

Won't fix.
